### DISCUSSION TITLE
Blending modes

### DISCUSSION THREAD

**Discussion by wiesehahn** [maplibre/maplibre#170] [Cross-project feature proposals]
Blending modes would be a really nice feature to boost visual appearance of maps and is already discussed in https://github.com/maplibre/maplibre-gl-js/issues/48

![image](https://user-images.githubusercontent.com/41429613/208449672-8bcd8196-c8a2-42ab-9194-8484f8423fca.png)

The image is from https://www.helenmakesmaps.com/post/how-to-use-blending-modes-in-mapping, which gives a nice overview.

**Comment by lseelenbinder** [comment]
I've hit this before when designing styles. Definitely would be cool to adjust the blending.

**Comment by bothness** [comment]
It would be great to have multiply blending in particular. I've previously resorted to having two synced maps on top of each other + css blending in order to achieve this, but would be much better if Maplibre could do it natively. Btw, this was a feature request for years on Mapbox GL, but was consistently ignored.

**Comment by wipfli** [comment]
Thanks for this suggestion. I think it would be best to open an issue in MapLibre GL JS and Native if there is not already one for this issue.

**Reply by wiesehahn** [reply]
https://github.com/maplibre/maplibre-gl-js/issues/48

**Comment by timbtimbtimb** [comment]
I'd love to see this feature too.

**Comment by npaquin47** [comment]
Blend mode Multiply on layer would be very useful for us.
